virt/virt-4-12-release-notes.adoc

```diff
@@ -100,9 +100,11 @@ The SVVP Certification applies to:
 === Installation
 
 
-//[id="virt-4-12-networking-new"]
-//=== Networking
+[id="virt-4-12-networking-new"]
+=== Networking
 
+//CNV-18349
+* You can now xref:../virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc#virt-about-services_virt-creating-service-vm[configure a load balancing service] by using the MetalLB Operator in layer 2 mode.
```

Comment:
```suggestion
* You can now xref:../virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc#virt-about-services_virt-creating-service-vm[configure a load balancing service] by using the MetalLB Operator.
```

@sbulage keep me honest here, but I believe we support both modes. Our integration in CNV is not affected by the selected mode.

Reply:
@phoracek TBH, it doesn't matter as MetalLB will available for not only CNV only but also for OpenShift too.